packages/backend-defaults/src/entrypoints/scheduler/lib/util.ts

```diff
@@ -18,6 +18,8 @@ import { InputError } from '@backstage/errors';
 import { Knex } from 'knex';
 import { DateTime, Duration } from 'luxon';
 
+export const TRACER_ID = '@backstage/backend-defaults/scheduler';
```

Comment:
This differs from the format in https://github.com/backstage/backstage/blob/b9379b45bb1b296377cb624b22eda32713c3a2df/plugins/catalog-backend/src/util/opentelemetry.ts#L19-L20, is that intentional?

Reply:
Yes it was. I was trying to match the pattern I saw in the broader otel instrumentation in JS. They all seem to use the package name as the trace id: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/6e8989de4d60c32bc9b3e3ef760e92c68ae7f491/plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts#L45-L48

The first two arguments to `super(...)` end up getting passed all the way down to `trace.getTracer(...)`. See: https://github.com/open-telemetry/opentelemetry-js/blob/d4035eb9733c101d53780033a4e58caebec73619/experimental/packages/opentelemetry-instrumentation/src/instrumentation.ts#L51-L71

I've been reading more into this https://opentelemetry.io/docs/specs/otel/trace/api/#get-a-tracer and I think that this might not have been quite right. The library reference pattern makes sense when in the context of an instrumentation library but that's not what we have here. I'll change things to match the pattern you pointed out.

Reply:
Done 🎉